libnavigation-core/src/main/java/com/mapbox/navigation/core/navigator/LocationEx.kt

```diff
@@ -48,11 +51,48 @@ internal fun Location.toFixLocation(): FixLocation {
         provider,
         bearingAccuracy,
         speedAccuracy,
-        verticalAccuracy
+        verticalAccuracy,
+        if (extras != null) extras.toMap() else Bundle().toMap()
     )
 }
 
 internal fun List<FixLocation>.toLocations(): List<Location> = this.map { it.toLocation() }
 
 private fun isCurrentSdkVersionEqualOrGreaterThan(sdkCode: Int): Boolean =
     Build.VERSION.SDK_INT >= sdkCode
+
+internal fun HashMap<String, Value>.toBundle(): Bundle? {
```

Comment:
It will be nice to have some alias for HashMap<String, Value> what do you think?